banzai/astrometry.py

```diff
@@ -30,6 +30,7 @@ def do_stage(self, image):
 
         # Short circuit
         if image_catalog is None:
+            image.header['WCSERR'] = (4, 'Error status of WCS fit. 0 for no error')
```

Comment:
This error message is used in 5+ places. It should either go in a constant at the top of the file, or we should take the opportunity to create an errors.py to centralize error messages (also a great way to standardize them).